addons/account/models/account_move.py

```diff
@@ -749,6 +749,13 @@ def init(self):
             expressions=['journal_id', 'company_id', 'date'],
             where="made_sequence_gap = TRUE",
         )  # used in <account.journal>._query_has_sequence_holes
+        create_index(
+            self.env.cr,
+            indexname='account_move_duplicate_bills_idx',
+            tablename='account_move',
+            expressions=['ref'],
+            where='move_type IN (\'in_invoice\', \'in_refund\')',
```

Comment:
```suggestion
            where="move_type IN ('in_invoice', 'in_refund')",
```
That is less ugly to me...